docs/compass/04-01-installation.md

```diff
@@ -133,94 +138,8 @@ Once Compass is installed, Runtime Agent will be configured to fetch the Runtime
 
 ### Local Minikube installation
 
-To install Compass and Runtime components on Minikube, follow these steps:
-
-1. Use the Kyma CLI to provision a Minikube cluster:
-
-    ```bash
-    kyma provision minikube
-    ```
-
-1. Prepare the Installation CR and overrides:
-    
-    ```bash
-    wget https://raw.githubusercontent.com/kyma-project/kyma/master/installation/resources/installer-cr-cluster-runtime.yaml.tpl
-    wget https://raw.githubusercontent.com/kyma-project/kyma/master/installation/resources/installer-config-local.yaml.tpl
-    cat <<EOF > disable-legacy-connectivity-override.yaml
-    apiVersion: v1
-    kind: ConfigMap
-    metadata:
-      name: disable-legacy-connectivity-override
-      namespace: kyma-installer
-      labels:
-        installer: overrides
-        kyma-project.io/installation: ""
-    data:
-      global.disableLegacyConnectivity: "true"
-    EOF
-    ```
-
-1. In order to reduce memory and CPU usage, from the downloaded `installer-cr-cluster-runtime.yaml.tpl` file, comment out the components you don't want to use, such as `monitoring`, `tracing`, `logging`, and `kiali`.
```

Comment:
This deleted note regarding the exclusion and inclusion of certain components seems reasonable to still exist in one form or another. If not for us, then for newer contributors who might look into streamlining their kyma installation for some reason